### DISCUSSION TITLE
where are the configs on the server?

### DISCUSSION THREAD

**Discussion by Xydozhnik11** [wg-easy/wg-easy#450] [General]
tell me please, how can i find  created config.conf on my server? in which directory?  i mean without web ui.      using   winscp

**Comment by eandersons** [comment]
With "on server" you mean host or container?

Either way it depends on how you created the WireGuard Easy container, but documentation suggests that bind-mounted volumes are used, so configuration files can be located in:

- if `docker run` was used to create the container: `.wg-easy` in directory where `docker run` was executed;
  command's `docker run` option `-v` specifies which directory should be mounted in the WireGuard Easy container;
- if Docker Compose was used to create the container: in the directory where `docker compose up --detach` was executed (look at the section `volumes` in `docker-compose.yml` to make sure which directory is supposed to be mounted in the container).

**Reply by hadi-dindar** [reply]
hello . do you know how we can add clients it would be nice to have statistics to this panel ?

**Comment by nevalashka** [comment]
> tell me please, how can i find created config.conf on my server? in which directory? i mean without web ui. using winscp

All your config files that you can see in web panel are included in one file located at: your-home-folder/.wg-easy/wg0.conf
